src/utils/tokens.js

```diff
@@ -0,0 +1,31 @@
+
+import sendJson from '../tmp_fetch_send_json'
+import { ACCOUNT_HELPER_URL } from './wallet'
+import * as Sentry from '@sentry/browser'
+import { wallet } from './wallet'
+
+export const getLikelyContracts = async (accountId) => (
+    await logAndIgnoreError(sendJson('GET', `${ACCOUNT_HELPER_URL}/account/${accountId}/likelyTokens`))
+)
+
+export const getMetadata = async (contract, accountId) => {
+    const account = await wallet.getAccountBasic(accountId)
+
+    let metadata = await logAndIgnoreError(account.viewFunction(contract, 'ft_metadata'))
+    return {
+        contract,
+        metadata
```

Comment:
Curious -- does `payload.metadata` include `contract` key? It would be nice to add a comment here referencing the spec that we expect this object shape to follow, if it's documented somewhere, or just describe the shape of the expected response if there is no spec that we can reference.

Reply:
>Curious -- does payload.metadata include contract key?

No. I'm not sure what is the possible object shape - @vgrichina where we can check that?

Reply:
There is fungible token standard defining all these details:
https://github.com/near/NEPs/tree/master/specs/Standards/FungibleToken

Probably makes sense to link to it from the implementation given there is clearly confusion over where to look this up.